chlorophyll/codeview.py

```diff
@@ -241,5 +246,27 @@ def place_forget(self) -> None:
 
     def destroy(self) -> None:
         for widget in self._frame.winfo_children():
-            tkinter.BaseWidget.destroy(widget)
-        tkinter.BaseWidget.destroy(self._frame)
+            BaseWidget.destroy(widget)
+        BaseWidget.destroy(self._frame)
+
+    def horizontal_scroll(
+        self,
+        first: str | float,
+        last: str | float,
+    ) -> "CodeView":
+        self._hs.set(first, last)
+
+    def vertical_scroll(
+        self,
+        first: str | float,
+        last: str | float,
+    ) -> "CodeView":
+        self._vs.set(first, last)
+        self._line_numbers.reload(self.cget("font"))
+
+    def scroll_line_update(
+        self,
+        event: Event | None = None,
+    ) -> "CodeView":
+        self.horizontal_scroll((xview := self.xview())[0], xview[1])
+        self.vertical_scroll((yview := self.yview())[0], yview[1])
```

Comment:
Walrus is good, but first, I want Chlorophyll to work on Py3.7 till July, and second, it's unnecessary in this case.
```suggestion
        self.horizontal_scroll(*self.xview())
        self.vertical_scroll(*self.yview())
```
Is this method needed at all?